bundler/spec/dependabot/bundler/file_updater_spec.rb

```diff
@@ -1709,27 +1762,23 @@
     end
   end
 
-  context "with bundler 2 support enabled" do
-    let(:updater) do
-      described_class.new(
-        dependency_files: dependency_files,
-        dependencies: dependencies,
-        credentials: [{
-          "type" => "git_source",
-          "host" => "github.com"
-        }],
-        repo_contents_path: repo_contents_path,
-        options: {
-          bundler_2_available: true
-        }
-      )
-    end
-
+  context "with bundler 2 support enabled", :bundler_v2_only do
     describe "updated_dependency_files" do
-      it "fails as the native helper is not yet implemented" do
-        expect { updater.updated_dependency_files }.
-          to raise_error(Dependabot::NotImplemented,
-                         "Bundler 2 adapter does not yet implement update_lockfile")
+      it "updates the dependencies" do
```

Comment:
Is this case already tested elsewhere for bundler v2? Could we ✂️ ?

Reply:
I think so, yes! ✨